vendor/github.com/keybase/client/go/protocol/keybase1/extras.go

```diff
@@ -1664,6 +1664,7 @@ func (u UserPlusKeysV2) ToUserVersion() UserVersion {
 	}
 }
 
+// Can return nil.
```

Comment:
That's a good suggestion. I hope you don't mind if I ignore it just to shuttle this PR along. I don't think these end up on the kbfs side of the wire